tests/cypress/e2e/api_helpers.ts

```diff
@@ -1,3 +1,6 @@
+// TODO: Make custom Cypress commands out of these
```

Comment:
would be best to split this commit into another pr, so matteo can review this.

Reply:
Honestly, I don't think it's essential. More of a quality of life improvement.

Reply:
fine for me to include this changes. but i think matteo is the current "code owner", so i don't want to approve/reject changes here, he should do this.

Reply:
Shure, I'll request his review as well.